app/views/v2/pkg/show.mustache

```diff
@@ -128,6 +128,9 @@ import 'package:{{package}}/{{library}}';
       <h3 class="title">License</h3>
       <p>{{& package.license_html}}</p>
     {{/package.license_html}}
+
+    <h3 class="title">More</h3>
+    Packages that depend on <a href="{{{search_deps_link}}}">{{package.name}}</a>
```

Comment:
I meant make the whole line a link. Or just `Packages`, maybe? Having just the pkg name as the link content is a bit weird. We're not linking to `pkg/foo`

Reply:
I'm not really fan of overtly long links, but let's give it a try...